nyaa/static/css/main.css

```diff
@@ -94,17 +94,23 @@ table.torrent-list thead th.sorting_desc:after {
 
 @media (max-width: 991px){
 	.col-md-5 {
-	    margin-left: 20px;
-	    margin-bottom: 10px;
+		margin-left: 20px;
+		margin-bottom: 10px;
+	}
+}
+
+@media (min-width: 992px){
+	.col-md-5 {
```

Comment:
I don't think `.col-md-5` is a good selector for this.
Maybe `.panel-body .col-md-5`?